src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,242 @@
+                        size === TabSize.Small ? "tw-text-sm" : "tw-text-md",
+                    ])}
+                >
+                    {tabs.map((tab, index) => {
+                        return (
+                            <button
+                                data-test-id="tab-item"
+                                role="tab"
+                                type="button"
+                                aria-selected={tab.id === activeItemId}
+                                aria-controls={`${tab.id}-content`}
+                                aria-hidden={tab.disabled}
+                                tabIndex={tab.id === activeItemId ? 0 : -1}
+                                id={index >= tabIndexLimit ? "" : `${tab.id}-btn`}
+                                className={merge([
+                                    "tw-group tw-relative tw-mx-0 tw-pb-5 tw-pt-2 tw-px-2 tw-w-max tw-h-10 tw-cursor-pointer tw-flex tw-items-center tw-justify-center tw-whitespace-nowrap",
+                                    tab.disabled && "tw-text-text-disabled",
+                                    tab.id === activeItemId && "tw-font-medium",
+                                    index >= tabIndexLimit && "tw-invisible",
+                                ])}
+                                key={tab.id}
+                                onClick={() => (!tab.disabled && onChange ? onChange(tab.id) : null)}
+                                onKeyDown={(event) => handleKeyboardTabChange(event)}
+                            >
+                                {tab.decorator}
+                                <span className="tw-mr-1 tw-ml-1.5">{tab.label}</span>
+                                {tab.badge && (
+                                    <Badge disabled={tab.disabled} style={tab.badge.style}>
+                                        {tab.badge.children}
+                                    </Badge>
+                                )}
+                                {tab.id === activeItemId && (
+                                    <motion.div
+                                        data-test-id="tab-active-highlight"
+                                        layoutId="underline"
+                                        className="tw-absolute tw-h-1 tw-bg-violet-60 tw-rounded-t-x-large tw-w-full tw-bottom-0"
+                                    />
+                                )}
+                                {tab.id !== activeItemId && !tab.disabled && (
+                                    <div className="group-hover:tw-absolute group-hover:tw-h-1 group-hover:tw-bg-grey-20 group-hover:tw-rounded-t-x-large group-hover:tw-w-full group-hover:tw-bottom-0" />
+                                )}
+                            </button>
+                        );
+                    })}
+                </div>
+                {isOverflowing && (
+                    // Remove rotate class when new Dots icon is available
```

Comment:
```suggestion
```
You can create a ClickUp ticket to follow up but it is also fine with the current icon

Reply:
Just added the ClickUp task.